app/index/controller.js

```diff
@@ -15,6 +15,12 @@ import { scheduleOnce } from 'ember-runloop'
 
 const { testing } = Ember
 
+const MODEL_NAME_MAPPING = {
+  report: 'reports',
+  absence: 'absences',
+  'public-holiday': 'holidays'
```

Comment:
It is not obvious here why public holiday is a hard coded string here and report and absence are not.

Reply:
not obvious for someone who doesn't know JS :-P

Reply:
(object property keys have to be valid identifiers in JS. Something containing dashes isn't: https://stackoverflow.com/questions/1661197/what-characters-are-valid-for-javascript-variable-names)